core/modules/ip_encap.cc

```diff
@@ -27,7 +27,7 @@ struct snobj *IPEncap::Init(struct snobj *arg [[maybe_unused]]) {
   return nullptr;
 };
 
-pb_error_t IPEncap::InitPb(const bess::pb::IPEncapArg &arg [[maybe_unused]]) {
+pb_error_t IPEncap::InitPb(const bess::pb::EmptyArg &) {
```

Comment:
Oh I was assuming the case where the new IPEncapArg does not have any "required" fields.